src/optimize/watch/optmzr_role.js

```diff
@@ -17,21 +17,33 @@
  * under the License.
  */
 
+import { resolve } from 'path';
+
 import WatchServer from './watch_server';
 import WatchOptimizer from './watch_optimizer';
+// import { DllCompiler } from '../dynamic_dll_plugin';
+import { fromRoot } from '../../utils'; // TODO: remove when merge with master
+import { WatchCache } from './watch_cache';
 
 export default async (kbnServer, kibanaHapiServer, config) => {
+  const log = (tags, data) => kibanaHapiServer.log(tags, data);
```

Comment:
This anonymous function might be unnecessary? Seems like you could pass `kibanaHapiServer.log` directly unless you're trying to hide other arguments from the WatchCache

Reply:
This was a thing that was already being done @joshdover . I guess it is because the 3rd argument for the hapiServer log function is the `timestamp` and maybe who first wrote this had the idea to hide it.  By my side I'm okay just passing the `kibanaHapiServer.log`, so I will change it 😃